src/applications/discharge-wizard/components/v2/ReviewPage.jsx

```diff
@@ -4,12 +4,22 @@ import { connect } from 'react-redux';
 import { VaButtonPair } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
```

Comment:
There are a few lint warnings about needing to add variables to props validation. I'd recommend adding these if you can.